glide.yaml

```diff
@@ -54,6 +54,7 @@ import:
   - package: github.com/mholt/archiver
   - package: github.com/evergreen-ci/shrub
   - package: github.com/evergreen-ci/pail
+  - package: gonum.org/v1/gonum
```

Comment:
this is also vendored in amboy so we should make sure we remove it from there (and also maybe use the same version?